react/ReadMore/ReadMore.demo.js

```diff
@@ -125,6 +131,32 @@ export default {
         }
       ]
     },
+    {
+      label: 'Color',
+      type: 'radio',
+      states: [
+        {
+          label: 'Default Background',
+          transformProps: props => ({
+            ...omit(props, 'backgroundColor')
+          })
+        },
+        {
+          label: 'Card',
+          transformProps: props => ({
+            ...props,
+            backgroundColor: 'card'
```

Comment:
should it be backgroundComponentName? Or I'm tired?

Reply:
Oh yeah good catch, I think it should be